src/box/memtx_tx.c

```diff
@@ -3309,6 +3403,30 @@ memtx_tx_inplace_gap_item_new(struct txn *txn)
 	return item;
 }
 
+/*
+ * Saves the given @a key in the @a short_key buffer of size @a short_key_size
+ * if it fits or allocates a new one on the @a txn's region. Returns the key
+ * length in the @a key_len output parameter.
+ */
+static char *
+memtx_tx_save_key(struct txn *txn, const char *key, uint32_t part_count,
```

Comment:
Nit: looks like this can be introduced as a refactoring of gap items in a separate patch.

Reply:
Done.